chain/x/zoracle/internal/keeper/oracle_script_test.go

```diff
@@ -94,8 +99,60 @@ func TestAddTooLongOracleScriptName(t *testing.T) {
 
 	owner := sdk.AccAddress([]byte("owner"))
 	tooLongName := "oracle_script"
+	description := "description"
 	code := []byte("code")
 
-	err = keeper.AddOracleScript(ctx, owner, tooLongName, code)
+	err = keeper.AddOracleScript(ctx, owner, tooLongName, description, code)
+	require.NotNil(t, err)
+}
+
+func TestAddTooLongOracleScriptDescription(t *testing.T) {
+	ctx, keeper := CreateTestInput(t, false)
+
+	_, err := keeper.GetOracleScript(ctx, 1)
+	require.NotNil(t, err)
+
+	// Set MaxNameLength to 5
+	keeper.SetMaxDescriptionLength(ctx, 5)
+
+	owner := sdk.AccAddress([]byte("owner"))
+	name := "oracle_script"
+	tooLongDescription := "description"
+	code := []byte("code")
+
+	err = keeper.AddOracleScript(ctx, owner, name, tooLongDescription, code)
+	require.NotNil(t, err)
+}
+
+func TestEditOracleScriptTooLongDescription(t *testing.T) {
```

Comment:
Can you rearrange the code for consistency?

- TestAddOracleScript
  - Success
  - TooLongCode
  - TooLongName
  - TooLongDescription
- TestEditOracleScript
  - ...

Same of Data source

Reply:
"Same of Data source"!!!